src/inference/src/system_conf.cpp

```diff
@@ -295,52 +302,86 @@ std::vector<std::vector<int>> get_proc_type_table() {
-                }
+        auto cur_stream_id = stream_id_per_coretype.find(cpu._cpu_mapping_table[i][CPU_MAP_CORE_TYPE]);
+        if (cur_stream_id != stream_id_per_coretype.end() && cpu._cpu_mapping_table[i][CPU_MAP_USED_FLAG] == NOT_USED) {
+            stream_processors[cur_stream_id->second].push_back(cpu._cpu_mapping_table[i][CPU_MAP_PROCESSOR_ID]);
+            stream_numa_node_ids[cur_stream_id->second] = cpu._cpu_mapping_table[i][CPU_MAP_SOCKET_ID];
+            cpu_ids.push_back(cpu._cpu_mapping_table[i][CPU_MAP_PROCESSOR_ID]);
+            if (stream_processors[cur_stream_id->second].size() ==
+                static_cast<size_t>(streams_info_per_coretype.at(cur_stream_id->first)[THREADS_PER_STREAM])) {
+                stream_id_per_coretype.at(cur_stream_id->first)++;
+                stream_num_per_coretype[cur_stream_id->first]++;
+            }
+            if (stream_num_per_coretype[cur_stream_id->first] >=
+                streams_info_per_coretype.at(cur_stream_id->first)[NUMBER_OF_STREAMS]) {
+                stream_id_per_coretype.erase(cur_stream_id);
             }
         }
     }
-    auto flatten_stream_ids =
-        std::accumulate(res_stream_ids.begin(),
-                        res_stream_ids.end(),
-                        decltype(res_stream_ids)::value_type{},
-                        [](std::vector<std::vector<int>>& pre, std::vector<std::vector<int>>& cur) {
-                            pre.insert(pre.end(), cur.begin(), cur.end());
-                            return pre;
-                        });
+    if (cpu_status > NOT_USED) {
+        set_cpu_used(cpu_ids, cpu_status);
+    }
 
-    return flatten_stream_ids;
+    OPENVINO_DEBUG << "[ threading ] cpu_mapping_table:";
+    for (size_t i = 0; i < cpu._cpu_mapping_table.size(); i++) {
+        OPENVINO_DEBUG << cpu._cpu_mapping_table[i][CPU_MAP_PROCESSOR_ID] << " "
+                       << cpu._cpu_mapping_table[i][CPU_MAP_SOCKET_ID] << " "
+                       << cpu._cpu_mapping_table[i][CPU_MAP_CORE_ID] << " "
+                       << cpu._cpu_mapping_table[i][CPU_MAP_CORE_TYPE] << " "
+                       << cpu._cpu_mapping_table[i][CPU_MAP_GROUP_ID] << " "
+                       << cpu._cpu_mapping_table[i][CPU_MAP_USED_FLAG];
+    }
+    OPENVINO_DEBUG << "[ threading ] proc_type_table:";
+    for (size_t i = 0; i < cpu._proc_type_table.size(); i++) {
+        OPENVINO_DEBUG << cpu._proc_type_table[i][ALL_PROC] << " " << cpu._proc_type_table[i][MAIN_CORE_PROC] << " "
+                       << cpu._proc_type_table[i][EFFICIENT_CORE_PROC] << " "
+                       << cpu._proc_type_table[i][HYPER_THREADING_PROC];
+    }
+    OPENVINO_DEBUG << "[ threading ] streams_info_table:";
+    for (size_t i = 0; i < streams_info_table.size(); i++) {
+        OPENVINO_DEBUG << streams_info_table[i][NUMBER_OF_STREAMS] << " " << streams_info_table[i][PROC_TYPE] << " "
+                       << streams_info_table[i][THREADS_PER_STREAM];
+    }
```

Comment:
The logs here are printed on one line for each table, which may be more readable.

Reply:
Thanks, done.